bindings/rust/evmc-declare/Cargo.toml

```diff
@@ -17,7 +17,7 @@ heck = "0.3.1"
 proc-macro2 = "1.0"
 syn = { version = "1.0", features = ["full"] }
 # For documentation examples
-evmc-vm = { path = "../evmc-vm" }
```

Comment:
Bumpversion's config seems to cover this already (given the version line above ^^).

Reply:
Verified with running `bumpversion`. It works.